unkeyed/unkey
# made "Create New Key" always accessible #2414

## What does this PR do?

Initial Problem: The "Create New Key" button was positioned below the entire list of API cards, which can be extensive. When trying to create a new root key with workspace-only permissions, users need to scroll to the bottom of the page. This creates an inconvenient user experience, especially when the list of APIs is long. Ideally, the button should be sticky and always visible at the bottom of the window, regardless of the length of the API list.

Solution: Made "Create New Key" button sticky , so there is no need to scroll to bottom.
Fixes # (#2414)
Fixes #2414 




## Type of change: added some tailwind classes and some DOM manipulation




## How should this be tested?

- Test : Go to Root keys, and create a new root key , see the button to Create new Key is sticky and there is no need to scroll to bottom

## Checklist

### Required


### Appreciated

- [x] If a UI change was made: 

https://github.com/user-attachments/assets/04982d87-459e-48f7-8cf4-107df97775ce
